cmd/qat_plugin/dpdkdrv/dpdkdrv.go

```diff
@@ -62,11 +64,54 @@ var qatDeviceDriver = map[string]string{
 	"6f55": "d15xxvf",
 }
 
+// swapBDF returns ["C", "B", "A"], when the given parameter is ["A", "B", "C"].
```

Comment:
just read this again and I don't think this comment is accurate. The order of `devstrings` does not change, right?